landingpage/src/components/AlterConfigForName.tsx

```diff
@@ -14,10 +14,12 @@ type AlterOptionKey =
 const AlterConfigForName: React.FC<{
 	queryString: string;
 	currentOwner: string;
+	onInscriptionChange: (output: string[]) => void;
 	// We might want to add later a config object.
-}> = ({ queryString, currentOwner }) => {
+}> = ({ queryString, currentOwner, onInscriptionChange }) => {
 	const [selectedOption, setSelectedOption] =
 		useState<AlterOptionKey>('website');
+	const [inscriptions, setInscriptions] = useState<string[]>(['']);
```

Comment:
There is only one line and one inscription per `AlterConfigForName`:
`const [inscription, setInscription] = useState<string>('');`